types/passport-saml/passport-saml-tests.ts

```diff
@@ -5,29 +5,29 @@ import MultiSamlStrategy = require('passport-saml/multiSamlStrategy');
 import fs = require('fs');
 
 const samlStrategy = new SamlStrategy.Strategy(
-    {
-        name: 'samlCustomName',
-        path: '/login/callback',
-        entryPoint: 'https://openidp.feide.no/simplesaml/saml2/idp/SSOService.php',
-        issuer: 'passport-saml',
-        cacheProvider: {
-            save(key: string, value: {}) {
-                // save the key with the optional value, invokes the callback with the value saves
-            },
-            get(key: string) {
-                // invokes 'callback' and passes the value if found, null otherwise
-            },
-            remove(key: string) {
-                // removes the key from the cache, invokes `callback` with the
-                // key removed, null if no key is removed
-            }
-        },
-        cert: fs.readFileSync('/path/to/cert.crt', 'UTF8')
-    },
-    (profile: {}, done: (err: Error | null, user: {}, info?: {}) => void) => {
-        const user = {};
-        done(null, user);
-    }
+	{
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/passport-saml/passport-saml-tests.ts;
  Line: 8;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.